slick/src/main/scala/slick/jdbc/meta/MColumn.scala

```diff
@@ -24,7 +24,7 @@ object MColumn {
           case _ => None
         }, r.<<, r.<<, r.skip.skip.<<, r.<<, DatabaseMeta.yesNoOpt(r),
         if(r.hasMoreColumns) MQName.optionalFrom(r) else None,
-        r.<<?,
+        r.<<?[Int],
```

Comment:
Wouldn't that mean it still crashes? I suppose we really need to read an `Any` here.